### ISSUE TITLE
[Question] The loss of finetuning

### ISSUE THREAD

**Issue by YuchenLiu98** [haotian-liu/LLaVA#256]
### Question

Hi, I just use the official released pretrained weight, and continue to finetune with instruction tuning data based on that pretrained weight. The initial loss is 10.375, which is much higher than your finetuning log with loss around1. Do you know why for this problem? Thanks a lot for your help.
